test/unit/endpoints/verification/verification.service.spec.ts

```diff
@@ -71,6 +74,25 @@ describe('Verification Service Unit Tests', () => {
     expect(verificationService).toBeDefined();
   });
 
+  it('shoul update genetic analyst status', async () => {
+    const NOW = 0;
+    const ACCOUNT_ID = 'ACCOUNT_ID';
+    const VERIFICATION_STATUS = 'Verified';
+
+    const RESULTS = await verificationService.verificationGeneticAnalyst(
```

Comment:
Remove unused `RESULTS`, you don't need to store the results of `verificationGeneticAnalyst` into the variable.